toolkit/scripts/toolchain.mk

```diff
@@ -169,10 +171,25 @@ $(final_toolchain): $(raw_toolchain) $(STATUS_FLAGS_DIR)/build_toolchain_srpms.f
 .SILENT: $(toolchain_rpms)
 
 ifeq ($(REBUILD_TOOLCHAIN),y)
+
+# Our manifest files should always track the contents of the freshly built archives exactly
+#   Currently non-blocking, to make this a blocking check switch to `print_error` instead of
+#   `print_warning`
+$(STATUS_FLAGS_DIR)/toolchain_verify.flag: $(TOOLCHAIN_MANIFEST) $(final_toolchain)
+	@echo Validating contents of toolchain against manifest...
+	tar -tzf $(final_toolchain) | sed 's+built_rpms_all/++g' | sed '/^$$/d' | sort > $(toolchain_actual_conents)
+	sort $(TOOLCHAIN_MANIFEST) > $(toolchain_expected_conents)
+	@diff="$$(comm -3 $(toolchain_actual_conents)  $(toolchain_expected_conents) )" && \
+	if [ -n "$${diff}" ]; then \
+		echo "ERROR: Missmatched packages between '$(TOOLCHAIN_MANIFEST)' and '$(final_toolchain)':" && \
+		echo "$${diff}"; \
```

Comment:
```suggestion
		@echo "ERROR: Missmatched packages between '$(TOOLCHAIN_MANIFEST)' and '$(final_toolchain)':" && \
		echo "$${diff}"; \
```

Reply:
The `@` on the `diff=...` should still be applying here. Any idea of a better way to do this though? It all needs to be one line with `\` to break it up or the `diff` variable doesn't persist since Make runs a instance of the shell program for every line otherwise.

Reply:
Sorry, yes, should be fixed now.